tasks1.js

```diff
@@ -0,0 +1,107 @@
+// 1
+function convertMinutesToHours(minutes) {
+  return (Math.floor(minutes / 60)) + ':' + (minutes % 60);
+}
+console.log(convertMinutesToHours(75));
+
+
+// 2
+function isTextUppercase(text) {
+  return /^[A-Z]*$/.test(text)
+}
+console.log(isTextUppercase('START'));
+
+
+// 3
+function isDivisible(n, x, y) {
+  if (n % x === 0 && n % y === 0) {
+    return true;
+  } else return false;
+}
```

Comment:
I know that codewars is using one letter arguments but we don't need to follow this and give a descriptive arguments names.

Also it can be done easier
```suggestion
function isDivisible(n, x, y) {
  return n % x === 0 && n % y === 0) 
  }
```